monai/networks/nets/densenet.py

```diff
@@ -296,14 +296,17 @@ class DenseNet121(DenseNet):
 
     def __init__(
         self,
+        spatial_dims: int,
+        in_channels: int,
+        out_channels: int,
         init_features: int = 64,
         growth_rate: int = 32,
         block_config: Sequence[int] = (6, 12, 24, 16),
         pretrained: bool = False,
         progress: bool = True,
         **kwargs,
     ) -> None:
-        super().__init__(init_features=init_features, growth_rate=growth_rate, block_config=block_config, **kwargs)
+        super().__init__(spatial_dims=spatial_dims, in_channels=in_channels, out_channels=out_channels, init_features=init_features, growth_rate=growth_rate, block_config=block_config, **kwargs)
         if pretrained:
             if kwargs["spatial_dims"] > 2:
```

Comment:
not with the current change because it breaks the pretrained=True use case. perhaps also update the following line it would work:
```suggestion
            if spatial_dims > 2:
```